azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineExtensionImageVersion.java

```diff
@@ -37,4 +38,10 @@
      * @return virtual machine extension image this version represents
      */
     VirtualMachineExtensionImage getImage();
+
+
+    /**
+     * @return an observable upon subscription emits the image
+     */
+    Observable<VirtualMachineExtensionImage> getImageAsync();
```

Comment:
new method on a previously released interface means need for `@Beta(SinceVersion.V1_2_0)`